### PR TITLE
Bot now fetches weather from owm API, added a new env var, bot prints…

### LINKED ISSUE #12: Get weather bot
The incoming message from Slack to your bot can be treated as an input string for many additional activities. One such activity is as follows. 
If the input string is the name of a city, then it can be sent to a 3rd party API such as Weather API (https://openweathermap.org/api). The Weather API then returns the current weather details (normally in JSON or XML format) and you can send them to your slack channel.

### PR DESCRIPTION
**Pull request by taimoorgit**
… usage on input err

Pair programming by Taimoor and Adam

Added a new environment variable

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,3 @@
 SIGNING_SECRET=
-SLACK_TOKEN=
\ No newline at end of file
+SLACK_TOKEN=
+OPEN_WEATHER_MAP=
```
Comment (Squishy123) [inline .env.example:3]: Will add this to heroku

**Review by JoseArnel — APPROVED** [review]
Looks Good

File: requirements.txt [hunk 2]
```diff
@@ -3,4 +3,5 @@ requests
 slack
 python-dotenv
 slackeventsapi
-slackclient
\ No newline at end of file
+slackclient
+pyowm
```
Comment (Squishy123) [inline requirements.txt:7]: cool!

**Review by Squishy123 — APPROVED** [review]
Approved! 👍

**Pull request merged**